internal/dbtest/query_test.go

```diff
@@ -943,3 +945,128 @@ func TestQuery(t *testing.T) {
 		}
 	})
 }
+
+func TestAlterTable(t *testing.T) {
+	needsAlterTable := needs(feature.AlterTableQuery, "ALTER TABLE")
+	type Model struct {
+		ID     int64
+		Old    string
+		Active bool
+	}
+
+	cases := []TestCase{
+		{
+			"rename column", needsAlterTable,
+			func(db *bun.DB) schema.QueryAppender {
+				return db.NewAlterTable().Model((*Model)(nil)).
+					RenameColumn().Column("old").To("new")
```

Comment:
Any reasons why not simply `RenameColumn("old_name", "new_name")`?

Reply:
Ditto: https://github.com/uptrace/bun/pull/726#discussion_r1044640112